device-api/src/config/mod.rs

```diff
@@ -316,6 +316,27 @@ mod tests {
             &["npu0pe0", "npu0pe1", "npu1pe0", "npu1pe1"],
         );
 
+        let config = "warboy(1)*1,npu:0:0".parse::<DeviceConfig>()?;
+        let found_device_files = find_device_files_in(&config, &devices_with_statuses)?;
+        let mut found_device_file_names: Vec<&str> =
+            found_device_files.iter().map(|f| f.filename()).collect();
+        found_device_file_names.sort();
+        assert_eq!(found_device_file_names, &["npu0pe0", "npu0pe1"]);
+
+        let config = "warboy(2)*1,npu:0:0".parse::<DeviceConfig>()?;
+        let found_device_files = find_device_files_in(&config, &devices_with_statuses)?;
+        let mut found_device_file_names: Vec<&str> =
+            found_device_files.iter().map(|f| f.filename()).collect();
+        found_device_file_names.sort();
+        assert_eq!(found_device_file_names, &["npu0pe0", "npu1pe0-1"]);
+
+        let config = "warboy(1)*1,npu:0:0-1".parse::<DeviceConfig>()?;
+        let found_device_files = find_device_files_in(&config, &devices_with_statuses)?;
+        let mut found_device_file_names: Vec<&str> =
+            found_device_files.iter().map(|f| f.filename()).collect();
+        found_device_file_names.sort();
+        assert_eq!(found_device_file_names, &["npu0pe0-1", "npu1pe0"]);
+
         Ok(())
```

Comment:
I added some tests, does it match the expected behavior for `find_device_files`?